src/swagger.ts

```diff
@@ -0,0 +1,159 @@
+'use strict';
+
+import * as vscode from 'vscode';
+
+export class Swagger {
+    constructor(definition: any, location: string) {
+        this.swagger = definition;
+        this.location = location;
+    }
+
+    private swagger: any = null;
+    private location: string = "";
+
+    public getExampleNames(path: string, method: string): string[] {
+        let examples: string[] = [];
+        let __this = this;
+
+        for (var name in this.swagger.paths[path][method]['x-ms-examples']) {
+            examples.push(__this.swagger.paths[path][method]['x-ms-examples'][name]['$ref']);
+        }
+
+        return examples;
+    }
+
+    public generateRestApiTasks(path: string, method: string, example: any): string {
+                        
+        let playbook: string = ""
+
+        let url: string = /*this.swagger['schemes'][0] + '://' + this.swagger['host'] + '/' +*/ path;
+        url = url.replace("{subscriptionId}", "{{ lookup('env','AZURE_SUBSCRIPTION_ID') }}");
+
+        method = method.toLowerCase();
+        playbook += //"# https://docs.microsoft.com/en-us/rest/api/" + area + "/" + resource + "/" + internalMethod + '\r';
+                    "- name: Call REST API - " + this.swagger.paths[path][method]['operationId'] + "\r" +
+                    "\t" + ((method == 'get') ? "azure_rm_resource_facts" : "azure_rm_resource") + ":\r" +
+                    "\t\t# url: " + url + "\r" + 
+                    "\t\tapi_version: '" + this.swagger['info']['version'] + "'\r";
+
+        if (method != 'get' && method != 'put' && method != 'delete') {
```

Comment:
didn't get the point, why not just method.toUpperCase() for all

Reply:
get / put / delete are default values

get is default for azure_rm_resource_facts
delete is default when state: absent
put is default for azure_rm_resource